doc/rst/technotes/libraries.rst

```diff
@@ -551,6 +554,140 @@ include the chapel runtime and standard modules for use in a non-Chapel program
 and when the library is linked to a Chapel program this leads to multiple
 definitions of these functions.
 
+Using Your Library in Multilocale Settings
+==========================================
+
+Prerequisites
+-------------
+
+Chapel supports ``--library`` when ``CHPL_COMM = gasnet``.  We intend to support
+other settings in the future, see :ref:`Other Settings` in the :ref:`Multilocale
+Caveats` section for more information.
+
+To compile a multilocale library, `ZeroMQ <http://zeromq.org/>`_ must be
+installed.
+
+If ZeroMQ is not installed in a way that enables your C compiler to find it
+easily, the environment variable ``CHPL_ZMQ_HOME`` can be set.  This environment
+variable should be set to a directory containing both an ``include`` directory
+which contains ``zmq.h`` and a ``lib`` directory which contains ``libzmq.*``.
+For example, for a directory structure:
+
+.. code-block:: text
+
+   |-- .local/
+   |    |-- include/
+   |    |    |-- zmq.h
+   |    |-- lib/
+   |    |    |-- libzmq.a
+   |    |    |-- libzmq.so
+
+``CHPL_ZMQ_HOME`` would be set to ``/absolute/path/to/.local/``.
+
+Initializing Your Multilocale Library
+-------------------------------------
+
+From the user's perspective, a strategy similar to that of single locale
```

Comment:
If users are reading this document... "From the user's perspective" might be the wrong way to word it.

I suggest this:

> Multilocale libraries can be used in a manner similar to single local libraries. However, as with
> transitioning between a single locale executable and a multilocale one, it is necessary number of locales required for the multilocale library.